tests/planner/ompl/test_OMPLSimplifier.cpp

```diff
@@ -0,0 +1,189 @@
+#include <aikido/planner/ompl/MotionValidator.hpp>
+#include <ompl/geometric/planners/rrt/RRTConnect.h>
+
+using StateSpace = aikido::statespace::dart::MetaSkeletonStateSpace;
+using aikido::planner::ompl::getSpaceInformation;
+using aikido::planner::ompl::CRRT;
+using aikido::planner::ompl::CRRTConnect;
+
+namespace {
+
+  template <typename T>
+  double computeTrajLength(T& traj,
+      aikido::statespace::dart::MetaSkeletonStateSpacePtr stateSpace,
+      aikido::distance::DistanceMetricPtr& dmetric)
+  {
+    auto stateCurrent = stateSpace->createState();
+    auto stateNext = stateSpace->createState();
+
+    double trajDistance = 0.0;
+    
+    for(size_t i = 0; i < traj->getDuration(); ++i)
+    {
+      traj->evaluate(i, stateCurrent);   
+      traj->evaluate(i+1, stateNext);
+      trajDistance += dmetric->distance(stateCurrent, stateNext);
+    }
+
+    return trajDistance;
+  }
+
+  aikido::trajectory::InterpolatedPtr planTrajForTest(
+      aikido::statespace::dart::MetaSkeletonStateSpacePtr _stateSpace,
+      aikido::statespace::InterpolatorPtr& _interpolator,
+      aikido::distance::DistanceMetricPtr& _dmetric,
+      aikido::constraint::SampleablePtr& _sampler,
+      aikido::constraint::TestablePtr& _collConstraint,
+      aikido::constraint::TestablePtr& _boundsConstraint,
+      aikido::constraint::ProjectablePtr& _boundsProjection,
+      Eigen::Vector3d _startPose, Eigen::Vector3d _goalPose          
+    )
+  {
+
+    auto startState = _stateSpace->createState();
+    auto subState1 = _stateSpace->getSubStateHandle<R3>(startState, 0);
+    subState1.setValue(_startPose);
+
+    auto goalState = _stateSpace->createState();
+    auto subState2 = _stateSpace->getSubStateHandle<R3>(goalState, 0);
+    subState2.setValue(_goalPose);
+
+    // Plan
+    auto traj = aikido::planner::ompl::planOMPL<ompl::geometric::RRTConnect>(
+        startState, goalState, _stateSpace, _interpolator, std::move(_dmetric),
+        std::move(_sampler), std::move(_collConstraint),
+        std::move(_boundsConstraint), std::move(_boundsProjection), 5.0, 0.1);
+
+    return traj;
+
+  }
+
+} // nampespace
+
+
+// Test that the start and goal positions do not change
+TEST_F(SimplifierTest, EndPointsRemainUnchanged)
+{
+  Eigen::Vector3d startPose(-5, -5, 0);
+  Eigen::Vector3d goalPose(5, 5, 0);
+
+  // Original Trajectory
+  auto traj = planTrajForTest(stateSpace, interpolator, dmetric, sampler, collConstraint, 
+      boundsConstraint, boundsProjection, startPose, goalPose);
+
+  // Setup for simplification
+  aikido::distance::DistanceMetricPtr dmetric = aikido::distance::createDistanceMetric(stateSpace);
+  aikido::constraint::SampleablePtr sampler = aikido::constraint::createSampleableBounds(stateSpace, make_rng()); 
+  aikido::constraint::ProjectablePtr boundsProjection = aikido::constraint::createProjectableBounds(stateSpace);
+  aikido::constraint::TestablePtr boundsConstraint = aikido::constraint::createTestableBounds(stateSpace);
```

Comment:
I won't need any of these anymore since I will no longer use planOMPL, the ones in `SetUp()` should suffice